api/types.proto

```diff
@@ -489,12 +489,17 @@ message NetworkAttachmentConfig {
 	// Target specifies the target network for attachment. This value must be a
 	// network ID.
 	string target = 1;
+	
 	// Aliases specifies a list of discoverable alternate names for the service on this Target.
 	repeated string aliases = 2;
+	
 	// Addresses specifies a list of ipv4 and ipv6 addresses
 	// preferred. If these addresses are not available then the
 	// attachment might fail.
 	repeated string addresses = 3;
+	
+	//Driver options is a map of driver specific options for the network target
```

Comment:
Can you please update the comment as requested above: https://github.com/docker/swarmkit/pull/2176/files#r115864626

This will make it consistent with the godoc-style comments elsewhere in this message that refer to the camelcased versions of the names.